src/extension.ts

```diff
@@ -588,6 +477,25 @@ export async function activate(context: vscode.ExtensionContext) {
 					const decoded = createPullRequestParamsCodec.decode(arg0);
 
 					if (decoded._tag === 'Right') {
+						const {
+							createNewBranch,
+							createPullRequest,
+							targetBranch,
+							stagedJobs,
+							commitMessage,
+						} = decoded.right;
+
+						const stagedJobHashes = new Set(
+							stagedJobs.map(({ hash }) => hash as JobHash),
+						);
+						messageBus.publish({
+							kind: MessageKind.acceptJobs,
+							jobHashes: stagedJobHashes,
+						});
+
+						// @TODO!!! temp workaround, currently no way to "await" for jobs to be applied to the file system
```

Comment:
what do you suggest as a solution? I guess we could await a message that confirms that jobs have been accepted?